script.js

```diff
@@ -0,0 +1,16 @@
+// Select Form
+const form = document.getElementById('resumeInputForm')
+
+// const nameInput = document.getElementById('formGroupNameInput')
+
+// let name = nameInput.value
+// console.log(nameInput)
+
+// Methods
+
+const handleFormSubmit = event => {
+  event.preventDefault()
+  const data = {}
```

Comment:
What were your thoughts on how to handle multiple inputs like jobs and education? I'm looking for suggestions

Reply:
Reference your e.target or e.target.value for working with multiple inputs - this way you can do conditional checks for handling different ways.

If you work with React, this becomes a lot easier! https://reactjs.org/docs/forms.html#handling-multiple-inputs